hackathon_site/hardware/serializers.py

```diff
@@ -39,6 +39,14 @@ def get_unique_hardware_count(obj: Category) -> int:
         return obj.hardware_set.annotate(Count("id", distinct=True)).count()
 
 
+class ChangeOrderStatusSerializer(serializers.ModelSerializer):
+    hardware_set = HardwareSerializer(many=True, read_only=True)
+
+    class Meta:
+        model = Order
+        fields = ("id", "hardware_set", "team", "status", "created_at", "updated_at")
```

Comment:
Just an early heads up - the only field we want users to be able to change via the API is `status` (`updated_at` should also be set by the backend, but not the API). You can set the rest as read only fields